src/trace-plugin-interface.js

```diff
@@ -209,10 +223,32 @@ PluginAPI.prototype.runInRootSpan = function(options, fn) {
 
 /**
  * Convenience method which obtains a Transaction object with getTransaction()
+ * and creates a new child span nested within this transaction. If there is
+ * no current Transaction object, this function returns null.
+ * @param {object} options An object that specifies options for how the child
+ * span is created and propogated. @see Transaction.prototype.createChildSpan
+ * @returns A new ChildSpan object, or null if there is no active root span.
+ */
+PluginAPI.prototype.createChildSpan = function(options, fn) {
```

Comment:
I agree.  With both `createChildSpan` and `runInChildSpan` present, one might expect one to be synchronous while the other is asynchronous, but `runInChildSpan` is synchronous (provided the supplied function is synchronous).